build.sh

```diff
@@ -111,6 +111,9 @@ else
 fi
 format_error_files=""
 
+# Ignore those PEP8 rules
+PEP8_IGNORES="E121,E123,E126,E226,E24,E704"
```

Comment:
My python3?-pep8 doesn't give me the defaults. Maybe I should pull up the script and see what the defaults actually are.